handlers/webhook-handler.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+class WebhookHandler {
+    constructor(models) {
+        this.model = models.subscriptionModel;
```

Comment:
assign models object itself here. and use `this.models.subscriptionModel` every place in this file.